src/_posts/platform/app/2000-01-01-force-https.md

```diff
@@ -23,14 +23,14 @@ Or, once a custom domain has been added:
 
 Activating the **Force HTTPS** feature enforces HTTPS access to all the domains
 attached to the application. It is achieved by activating a permanent
-redirection from HTTP to HTTPS and injecting the HSTS header in HTTPS responses.
+redirection from HTTP to HTTPS and injecting the `Strict-Transport-Security` header (abbreviated HSTS) in HTTPS responses.
 
 ## Force HTTPS a Scalingo Application
 
 Scalingo eases the process for you to enable "Force HTTPS" on your application.
-Head to the Settings tab of your web dashboard and check "Force HTTPS":
+Head to the Settings tab of your web dashboard, in the Routing section, check "Force HTTPS":
 
-{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_dashboard_settings.png" %}
+{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_dashboard_force_https.png" %}
```

Comment:
I updated the screenshot in the CDN. After this PR merge, I will remove screenshot_dashboard_settings.png from the CDN.